### ISSUE TITLE
`zh-cn` translations will never be compiled

### ISSUE THREAD

**Issue by nightire** [jasonmit/ember-intl#420]
- [x] I am on the [latest](https://github.com/jasonmit/ember-intl/releases/latest) ember-intl version
- [x] I have searched the [issues](https://github.com/jasonmit/ember-intl/issues) of this repo and believe that this is not a duplicate

Hi, sorry to bother again.

I found `translations/zh-cn.yaml` will never be compiled, I do check the supported locales and indeed `zh-cn` isn't exists in this list. But according to this earlier issue: https://github.com/andyearnshaw/Intl.js/issues/16, they have claimed `zh-hans` will fallback to `zh-cn`, so I'm not sure if ember-intl does fully supports this fallback?

**Comment by jasonmit** [comment]
Did you make sure to load the Intl polyfill data for the `zh-Hans-CN` locale?  Since you don't have translations for this, that locale data will not be bundled into the app code.

You can see a full list of all the locales supported by the Intl API polyfill here: https://github.com/andyearnshaw/Intl.js/tree/master/locale-data/jsonp

Here is how you can manually include locale data for particular locales your application needs to support:
https://github.com/jasonmit/ember-intl/blob/master/docs/intljs-polyfill.md#intljs-polyfill

**Comment by nightire** [comment]
@jasonmit OK, things get a little more complex, let me sort out:

1. yes, I know the supported API polyfill, and `zh-cn` is not in the list, although https://github.com/andyearnshaw/Intl.js/issues/16 said they provide fallback, but I don't see / know how it works.

2. yes, I know how to include locale data, but my problem is `zh-cn.js` doesn't exists in `/assets/intl/locales` path. BTW, the CLDR Data Full List has been moved to https://github.com/unicode-cldr/cldr-localenames-full/tree/master/main

3. After reading Unicode CLDR, I've realized that 'zh-cn' is not a valid locale name, so maybe the best way is to change our backend service to return `zh-hans` instead, which solves both 1. and 2.

4. I just don't understand:

    1. Why 'zh-cn' is not supported by Unicode CLDR, because the majority of Chinese People knows 'zh-cn' much more than 'zh-hans'. But I believe Unicode CLDR has its own reason, so I think ember-intl doesn't has to be take the responsibility

    2. the API polyfill and CLDR Data is NOT THE SAME THING, right? so even API polyfill provide a fallback to 'zh-cn' doesn't mean I can get a 'zh-cn' CLDR data, right? I wish I totally understand them this time, because their relationships are too complex, drove me nuts.

Anyway, ember-intl is awesome, and you always responds me very fast, thank you always.

It's not a bug, the only thing I have to do is use 'zh-hans' to replace 'zh-cn', simple move will fix everything.
